gulp/tasks/minifyCss.js

```diff
@@ -1,10 +1,9 @@
 var gulp      = require('gulp');
 var config    = require('../config');
 var postcss   = require('gulp-postcss');
-var autoprefixer = require('autoprefixer-core');
+var autoprefixer = require('gulp-autoprefixer');
```

Comment:
Because then you'd have to do `autoprefixer.process(cssstring).css`, not sure that is easily achieved in gulp (I'm only used to grunt).